modules/light-clients/08-wasm/types/export_test.go

```diff
@@ -1,16 +1,24 @@
 package types
 
+import (
+	storetypes "cosmossdk.io/store/types"
+)
+
 /*
 	This file is to allow for unexported functions and fields to be accessible to the testing package.
 */
 
 // MaxWasmSize is the maximum size of a wasm code in bytes.
 const MaxWasmSize = maxWasmSize
 
-// these fields are exported aliases for the payload fields passed to the wasm vm.
-// these are used to specify callback functions to handle specific queries in the mock vm.
-type (
-	// CallbackFn types
-	QueryFn = queryFn
-	SudoFn  = sudoFn
-)
```

Comment:
I've removed the usage of these in my test in another PR but forgot to remove them from `export_test.go`. So I'm doing this here